core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeExpressionBuilder.scala

```diff
@@ -572,4 +572,9 @@ class SnowflakeExpressionBuilder()
   override def visitExprSubquery(ctx: ExprSubqueryContext): ir.Expression = {
     ir.ScalarSubquery(ctx.subquery().accept(new SnowflakeRelationBuilder))
   }
+
+  override def visitExprJoin(ctx: ExprJoinContext): ir.Expression = {
+    ir.JoinMarkExpression(ctx.expr().accept(this))
+
```

Comment:
remove blank line

Reply:
done, will take care of it. Its work in progress and a draft PR, As a practice I commit the dev code on regular basis.